CMake/Platforms/WindowsCache.cmake

```
@@ -52,6 +52,7 @@ if(NOT UNIX)
     set(HAVE_GETHOSTNAME 1)
     set(HAVE_LIBZ 0)
 
+    set(HAVE_ARC4RANDOM 0)
     set(HAVE_ARPA_INET_H 0)
```

```suggestion
    set(HAVE_ARPA_INET_H 0)
    set(HAVE_FNMATCH 0)
    set(HAVE_SCHED_YIELD 0)
```
We also cache these for Windows, they are not present there.